src/Game.cpp

```diff
@@ -178,25 +180,25 @@ void Game::update_resolution() {
 	float new_ratio = (1.0 * width) / height;
 	float offset_x, offset_y, w, h;
 
-	if(original_ratio < new_ratio){
+	if (original_ratio < new_ratio) {
 		h = height; w = height * original_ratio;
 		offset_y = 0;
 		offset_x =  (720 / h) * (w - width) / 2;
-	}else{
+	} else {
 		w = width; h = width / original_ratio;
 		offset_x = 0;
 		offset_y = (1280 / w) * (h - height) / 2;
 	}
-
+                                                                                
```

Comment:
Trailing spaces are not allowed, just leave a blank line (without trailing spaces).